move_to_map_pokemon.py was broken in the last 24 hrs

I just did a dev pull this morning and now I get this issue when using 
"type": "MoveToMapPokemon",

I have yet to switch to the new walk speed Min/Max settings,but doubt that is the issue 

also pulled latest  PokemonGo-Map after this error 

Traceback (most recent call last):
  File "pokecli.py", line 626, in <module>
    main()
  File "pokecli.py", line 104, in main
    bot.tick()
  File "/DATA/PokemonGo-Bot/pokemongo_bot/__init__.py", line 498, in tick
    if worker.work() == WorkerResult.RUNNING:
  File "/DATA/PokemonGo-Bot/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 248, in work
    pokemon_list = self.get_pokemon_from_map()
  File "/DATA/PokemonGo-Bot/pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 147, in get_pokemon_from_map
    if pokemon['dist'] > ((pokemon['expire'] - now) * mean_walk_speed) and not self.config['snipe']:
KeyError: 'expire'
2016-08-17 00:12:46,012 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/raven/transport/threaded.py", line 174, in send_sync
    super(ThreadedHTTPTransport, self).send(data, headers)
  File "/usr/local/lib/python2.7/dist-packages/raven/transport/http.py", line 47, in send
    ca_certs=self.ca_certs,
  File "/usr/local/lib/python2.7/dist-packages/raven/utils/http.py", line 66, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 494, in open
    response = self._open(req, data)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 512, in _open
    '_open', req)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 466, in _call_chain
    result = func(*args)
  File "/usr/local/lib/python2.7/dist-packages/raven/utils/http.py", line 46, in https_open
    return self.do_open(ValidHTTPSConnection, req)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 1284, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/usr/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 895, in _send_output
    msg += message_body
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-17 00:12:46,017 [sentry.errors.uncaught] [ERROR] [u"KeyError: 'expire'", u'  File "pokecli.py", line 626, in <module>', u'  File "pokecli.py", line 104, in main', u'  File "pokemongo_bot/__init__.py", line 498, in tick', u'  File "pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 248, in work', u'  File "pokemongo_bot/cell_workers/move_to_map_pokemon.py", line 147, in get_pokemon_from_map']

--- a/pokemongo_bot/__init__.py
+++ b/pokemongo_bot/__init__.py
@@ -418,7 +418,7 @@ class PokemonGoBot(object):
         self.event_manager.register_event(
             'arrived_at_cluster',
             parameters=(
-                'forts', 'radius'
+                'num_points', 'forts', 'radius'
             )
         )
 

--- a/pokemongo_bot/cell_workers/follow_cluster.py
+++ b/pokemongo_bot/cell_workers/follow_cluster.py
@@ -39,7 +39,7 @@ class FollowCluster(BaseTask):
 
             if not self.is_at_destination:
                 msg = log_lure_avail_str + (
-                    "Move to destination {num_points}. {forts} "
+                    "Move to destination: {num_points} {forts} "
                     "pokestops will be in range of {radius}. Walking {distance}m."
                 )
                 self.emit_event(
@@ -49,7 +49,7 @@ class FollowCluster(BaseTask):
                         'num_points': cnt,
                         'forts': log_lured_str,
                         'radius': str(self.radius),
-                        'distance': str(distance(self.bot.position[0], self.bot.position[1], lat, lng))
+                        'distance': str(round(distance(self.bot.position[0], self.bot.position[1], lat, lng), 2))
                     }
                 )
 

--- a/pokemongo_bot/cell_workers/follow_cluster.py
+++ b/pokemongo_bot/cell_workers/follow_cluster.py
@@ -39,7 +39,7 @@ class FollowCluster(BaseTask):
 
             if not self.is_at_destination:
                 msg = log_lure_avail_str + (
-                    "Move to destination: {num_points} {forts} "
+                    "Move to cluster: {num_points} {forts} "
                     "pokestops will be in range of {radius}. Walking {distance}m."
                 )
                 self.emit_event(
@@ -71,9 +71,10 @@ class FollowCluster(BaseTask):
             elif not self.announced:
                 self.emit_event(
                     'arrived_at_cluster',
-                    formatted="Arrived at cluster. {forts} are in a range of {radius}m radius.",
+                    formatted="Arrived at cluster. {num_points} {forts} pokestops are in a range of {radius}m radius.",
                     data={
-                        'forts': str(cnt),
+                        'num_points': cnt,
+                        'forts': log_lured_str,
                         'radius': self.radius
                     }
                 )

--- a/pokemongo_bot/cell_workers/follow_cluster.py
+++ b/pokemongo_bot/cell_workers/follow_cluster.py
@@ -21,9 +21,9 @@ class FollowCluster(BaseTask):
         log_lure_avail_str = ''
         log_lured_str = ''
         if self.lured:
-            log_lured_str = 'lured '
             lured_forts = [x for x in forts if 'active_fort_modifier' in x]
             if len(lured_forts) > 0:
+                log_lured_str = 'lured '
                 self.dest = find_biggest_cluster(self.radius, lured_forts, '9QM=')
             else:
                 log_lure_avail_str = 'No lured pokestops in vicinity. Search for normal ones instead. '

--- a/pokemongo_bot/cell_workers/follow_cluster.py
+++ b/pokemongo_bot/cell_workers/follow_cluster.py
@@ -22,9 +22,9 @@ class FollowCluster(BaseTask):
         log_lured_str = ''
         if self.lured:
             log_lured_str = 'lured '
-            lured_forts = [x for x in forts if 'lure_info' in x]
+            lured_forts = [x for x in forts if 'active_fort_modifier' in x]
             if len(lured_forts) > 0:
-                self.dest = find_biggest_cluster(self.radius, lured_forts, 'lure_info')
+                self.dest = find_biggest_cluster(self.radius, lured_forts, '9QM=')
             else:
                 log_lure_avail_str = 'No lured pokestops in vicinity. Search for normal ones instead. '
                 self.dest = find_biggest_cluster(self.radius, forts)
@@ -39,7 +39,7 @@ class FollowCluster(BaseTask):
 
             if not self.is_at_destination:
                 msg = log_lure_avail_str + (
-                    "Move to destiny {num_points}. {forts} "
+                    "Move to destination {num_points}. {forts} "
                     "pokestops will be in range of {radius}. Walking {distance}m."
                 )
                 self.emit_event(

--- a/pokemongo_bot/cell_workers/move_to_map_pokemon.py
+++ b/pokemongo_bot/cell_workers/move_to_map_pokemon.py
@@ -144,7 +144,7 @@ class MoveToMapPokemon(BaseTask):
 
             # pokemon not reachable with mean walking speed (by config)
             mean_walk_speed = (self.bot.config.walk_max + self.bot.config.walk_min) / 2
-            if pokemon['dist'] > ((pokemon['expire'] - now) * mean_walk_speed) and not self.config['snipe']:
+            if pokemon['dist'] > ((pokemon['disappear_time'] - now) * mean_walk_speed) and not self.config['snipe']:
                 continue
 
             pokemon_list.append(pokemon)

--- a/pokemongo_bot/cell_workers/utils.py
+++ b/pokemongo_bot/cell_workers/utils.py
@@ -218,7 +218,8 @@ def find_biggest_cluster(radius, points, order=None):
     for point in points:
             if order is '9QM=':
                 #is a lure module - 9QM=
-                remaining = now = int(time.time()) - point['last_modified_timestamp_ms']
+                now = int(time.time())
+                remaining = now - point['last_modified_timestamp_ms']
                 f = point['latitude'], point['longitude'], remaining
             else:
                 f = point['latitude'], point['longitude'], 0

--- a/pokemongo_bot/cell_workers/utils.py
+++ b/pokemongo_bot/cell_workers/utils.py
@@ -216,8 +216,10 @@ def rad2deg(rad):
 def find_biggest_cluster(radius, points, order=None):
     graph = nx.Graph()
     for point in points:
-            if order is 'lure_info':
-                f = point['latitude'], point['longitude'], point['lure_info']['lure_expires_timestamp_ms']
+            if order is '9QM=':
+                #is a lure module - 9QM=
+                remaining = now = int(time.time()) - point['last_modified_timestamp_ms']
+                f = point['latitude'], point['longitude'], remaining
             else:
                 f = point['latitude'], point['longitude'], 0
             graph.add_node(f)

